src/lib/functions/server.ts

```diff
@@ -269,24 +269,20 @@ interface GetFunctionsServerOptions {
 
 const getFunctionsServer = (options: GetFunctionsServerOptions) => {
   const { functionsRegistry, siteUrl } = options
-  const app = express()
+  const app = new App()
   const functionHandler = createHandler(options)
 
-  app.set('query parser', 'simple')
-
-  app.use(
-    express.text({
-      limit: '6mb',
-      type: ['text/*', 'application/json'],
-    }),
-  )
-  app.use(express.raw({ limit: '6mb', type: '*/*' }))
+  app.use((req, res, next) => {
+    if (req.is('text/*') || req.is('application/json')) {
+      text({
+        payloadLimit: 6_291_456,
+      })(req, res, next)
+    }
+    return next?.()
+  })
+  app.use(raw({ payloadLimit: 6_291_456 }))
   app.use(createFormSubmissionHandler({ functionsRegistry, siteUrl }))
-  app.use(
-    expressLogging(console, {
-      blacklist: ['/favicon.ico'],
```

Comment:
Are we losing this behaviour?

Reply:
the blacklisting of the logging yes - I can add it to `@tinyhttp/logger`